packages/lib-classifier/src/store/tasks/drawingTools/Point.js

```diff
@@ -0,0 +1,12 @@
+import { types, onSnapshot } from 'mobx-state-tree'
+
+const Point = types.model('Point', {
+  color: types.optional(types.string, '#00ff00'),
```

Comment:
Do we want to code colours into the tools, or look them up from a shared lookup table?

Reply:
Shared lookup table sounds better, but maybe I'm not thinking about this quite right. And it's subject to change if we change how the tools are built in the lab. Hm, maybe for now I just change to `types.maybe(types.string)` and can revisit.

Reply:
I think `types.optional(types.string, '')` might be better in this case? It depends on if `color` can ever be null or undefined.